R/adl.R

```diff
@@ -122,4 +121,160 @@ adl_fun <- function (ADL_01, ADL_02, ADL_03, ADL_04, ADL_05) {
            "NA(b)"
            )
 
-}
\ No newline at end of file
+}
+
+#' @title The number of activities of daily living tasks that require help.
+#'
+#' @description A 6 category variable (ADL_score_5) representing the number of
+#'  activities of daily living tasks that require help. The following five ADL
+#'  variables are used. All variables are common across CCHS cycles from 2001
+#'  to 2014:
+#' \enumerate{
+#'   \item ADL_01 - Needs help preparing meals
+#'   \item ADL_02 - Needs help getting to appointments/errands
+#'   \item ADL_03 - Needs help doing housework
+#'   \item ADL_04 - Needs help doing personal care
+#'   \item ADL_05 - Needs help moving inside house
+#' }
+#' There are two categories to represent "missing" and "not applicable". 
+#'
+#' @param ADL_01 Needs help preparing meals
+#' @param ADL_02 Needs help getting to appointments/errands
+#' @param ADL_03 Needs help doing housework
+#' @param ADL_04 Needs help doing personal care
+#' @param ADL_05 Needs help moving inside house
+#'
+#' @return A derived variable (ADL_score_5) with 6 categories:
+#' \enumerate{
+#'   \item 0 - Needs help with 0 tasks
+#'   \item 1 - Needs help with at least 1 task
+#'   \item 2 - Needs help with at least 2 tasks
+#'   \item 3 - Needs help with at least 3 tasks
+#'   \item 4 - Needs help with at least 4 tasks
+#'   \item 5 - Needs help with at least 5 tasks
+#'   \item NA(a) - Not applicable. At least one of the component variables
+#'    had a value of not applicable
+#'   \item NA(b) - Missing. At least one of the component variables had a value
+#'    of missing, but none of had a **not applicable** value
+#' }
+#'
+#' @export
```

Comment:
Could you move the export after the examples so its easier to see which functions are exported